### ISSUE TITLE
Learning rate passed via OptimWrapper not updated in Learner

### ISSUE THREAD

**Issue by dnth** [fastai/fastai#3337]
**Versions**
```
fastai     : 2.3.0
fastcore   : 1.3.19
torch      : 1.7.1
```

**Describe the bug**
I ran the Imagenette notebook provided in Fastbook and specified the optimizer using OptimWrapper with a learning rate value. However the training does not use the lr value provided and defaults to 0.001
 
```
def get_learner(m):
    opt_func = partial(OptimWrapper, opt=torch.optim.SGD, lr=0.002)
    return Learner(dls, m, loss_func=nn.CrossEntropyLoss(), metrics=accuracy, opt_func=opt_func).to_fp16()
```
```
learn = get_learner(get_model())
learn.lr
```
Output
`0.001
`

**To Reproduce**
Load and run this Colab notebook: https://colab.research.google.com/drive/1sCtiSCSZ2CWDFWGP8yO8NYdKL5GEAjcu?usp=sharing

**Expected behavior**
`Learner` uses the lr passed via `OptimWrapper`.

**Comment by Salehbigdeli** [comment]
You can set the learning rate in the `Learner`. Is there any reason for setting the `lr` in your optimizer? I'm asking this becase solving this issue makes library more complex!

```python
def get_learner(m):
    opt_func = partial(OptimWrapper, opt=torch.optim.SGD)
    return Learner(dls, m, loss_func=nn.CrossEntropyLoss(), metrics=accuracy, opt_func=opt_func, lr=0.002).to_fp16()
```

**Comment by dnth** [comment]
https://github.com/fastai/fastai/pull/3365
